__tests__/cmds/openapi/index.test.ts

```diff
@@ -59,6 +60,7 @@ describe('rdme openapi', () => {
   beforeEach(() => {
     consoleInfoSpy = vi.spyOn(console, 'info').mockImplementation(() => {});
     consoleWarnSpy = vi.spyOn(console, 'warn').mockImplementation(() => {});
```

Comment:
I've always disliked having to do this and as oclif has some tooling in `captureOutput` to capture logs it. might be if we could move away from this somehow. Looks like we use the logging library to handle how logging happens in GHA but maybe those could be encapsulated into our `BaseCommand`?

Might be too big of a lift just for the sake of eliminating some unit test spites though.

Reply:
yeah def agreed — i have some follow-ups in my internal notes (that i'll document as proper tickets) to get rid of our `lib/logger` entirely in favor of oclif's logging but the refactor would be pretty involved and blow up the diff even further so i'll try and get that done in a follow-up PR at some point